app/dao/notifications_dao.py

```diff
@@ -377,6 +378,22 @@ def delete_notifications_created_more_than_a_week_ago_by_type(notification_type)
 
 
 @statsd(namespace="dao")
+def delete_notification_to_email_reply_to_more_than_a_week_ago():
+    seven_days_ago = date.today() - timedelta(days=7)
+
+    subq = db.session.query(Notification.id).filter(
+        func.date(Notification.created_at) < seven_days_ago
+    ).subquery()
+    deleted = db.session.query(
+        NotificationEmailReplyTo
+    ).filter(
+        NotificationEmailReplyTo.notification_id.in_(subq)
+    ).delete(synchronize_session='fetch')
+    db.session.commit()
+    return deleted
```

Comment:
Might be worth checking the subq length = the deleted length and logging if there is a difference.

or do a synchronize_session='evaluate' check the number match then run the actual delete?

Reply:
Will add this to `delete_notifications_created_more_than_a_week_ago_by_type` function to limit impact when there is an issue when deleting records